server/server/src/scheduled_tasks.rs

```diff
@@ -0,0 +1,32 @@
+use service::service_provider::ServiceProvider;
+use std::{sync::Arc, time::Duration};
+use tokio::task::JoinHandle;
+
+static TASK_INTERVAL: Duration = Duration::from_secs(10);
```

Comment:
I think we should maybe bump this up to 60s or something for now? Especially since we're spamming the logs on every run...

Reply:
Although it is really nice to have it respond quickly when testing emails, so I'm ok with leaving this!

Reply:
Added it as a config option? What do you reckon? [532aa0e](https://github.com/msupply-foundation/open-msupply/pull/5915/commits/532aa0e50ef60811d30793367cad21251ac75b57)

Reply:
Nice, that's handy, and good for support to be able to tweak it too.